README.md

```diff
@@ -58,12 +60,12 @@ test/
 
 ### CUDA Requirements
 
-Currently CUDA 10.2 is supported by the recipes in Open-CE. Please see [`doc/README.cuda_support.md`](doc/README.cuda_support.md) for details on setting
+Currently CUDA 10.2 and 11.0 are supported by the recipes in Open-CE. Please see [`doc/README.cuda_support.md`](doc/README.cuda_support.md) for details on setting
 up a proper build enviornment for CUDA support.
 
 Open-CE expects the `CUDA_HOME` environment variable to be set to the location of the CUDA installation.
 
-When building packages that use CUDA, a tar package of TensorRT 7.0 for CUDA 10.2 will need to be [downloaded](https://developer.nvidia.com/nvidia-tensorrt-7x-download) ahead of time. The downloaded file should be placed in a new local directory called `local_files`. The [cuda README](doc/README.cuda_support.md) has more information.
+When building packages that use CUDA, a tar package of TensorRT 7.0 for the intended CUDA version will need to be [downloaded](https://developer.nvidia.com/nvidia-tensorrt-7x-download) ahead of time. The downloaded file should be placed in a new local directory called `local_files`. The [cuda README](doc/README.cuda_support.md) has more information.
```

Comment:
I think we should write `a tar package of TensorRT 7.x for the ...`

Reply:
I changed this to just say Tensorrt. then I added the version in the Tensorrt table here:
https://github.com/open-ce/open-ce/pull/243/files#diff-cb96e2b228ce0a549bef55d3963cc18ee6540ecef9c6b85bc4cd6312c7acc108R123